### PR TITLE
Cat 2483 refacturing drone, adding move bricks for JumpingSumo Drone

### PR DESCRIPTION
**Pull request by cpozvek**

### CODE REVIEW

**Comment by 84n4n4** [comment]
Please run espresso tests

**Comment by cpozvek** [comment]
I did that, but my tests worked. So where is the problem?

**Comment by 84n4n4** [comment]
that is the trigger phase to have jenkins run the espresso tests....

File: catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/JumpingSumoMoveBackwardBrickTest.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+	@Category({Cat.AppUi.class, Level.Smoke.class, Cat.Gadgets.class})
+	@Test
+	@Flaky
+	public void jumpingSumoRotateLeftTest() {
+		int secondsValue = 3;
+		int powerValue = 40;
+
+		onBrickAtPosition(0).checkShowsText(R.string.brick_when_started);
+		onBrickAtPosition(brickPosition).checkShowsText("Move Jumping Sumo backward");
```
Comment (84n4n4) [inline catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/JumpingSumoMoveBackwardBrickTest.java]: please no hardcoded strings, pull them from R.string....

Comment (cpozvek) [reply catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/JumpingSumoMoveBackwardBrickTest.java]: ok, will change that

Comment (84n4n4) [reply catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/JumpingSumoMoveBackwardBrickTest.java]: can you also update the already merged tests to have them use R.string?

Comment (cpozvek) [reply catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/JumpingSumoMoveBackwardBrickTest.java]: yes, no problem

**Comment by cheidenreich** [comment]
please run espresso tests

File: catroid/build.gradle [hunk 2]
```diff
@@ -70,6 +70,7 @@ android {
     packagingOptions {
         exclude 'META-INF/maven/com.google.guava/guava/pom.properties'
         exclude 'META-INF/maven/com.google.guava/guava/pom.xml'
+        exclude 'META-INF/INDEX.LIST'
```
[the remark this answers is not in the document]
Comment (cpozvek) [reply catroid/build.gradle]: this is also needed for the parrot arsdk, we have compile problems if this is not excluded.

**Comment by cheidenreich** [comment]
please fix this conflict, then I'll merge this

**Comment by cpozvek** [comment]
fixed

**Comment by cheidenreich** [comment]
please run espresso tests

**Pull request merged**